gcloud-java-compute/src/test/java/com/google/gcloud/compute/ComputeImplTest.java

```diff
@@ -117,6 +119,37 @@
   private static final LicenseId LICENSE_ID = LicenseId.of("project", "license");
   private static final Boolean CHARGES_USE_FEE = true;
   private static final License LICENSE = new License(LICENSE_ID, CHARGES_USE_FEE);
+  private static final Operation.OperationError OPERATION_ERROR1 =
```

Comment:
Nit: could we import Operation.Operation[Error|Warning] to avoid having to specify "Operation."?